src/components/layout/layout.tsx

```diff
@@ -8,6 +9,17 @@ import state from '../../store';
 })
 export class Layout {
   @Prop() hasBack: boolean = true;
+  @State() shouldHide: boolean = false;
+
+  componentDidLoad() {
+    Keyboard.addListener('keyboardWillShow', () => {
```

Comment:
@charlesoma just a good practice, I suggest to remove listners in [disconnectedCallback()](https://stenciljs.com/docs/component-lifecycle#disconnectedcallback) to prevent potential mem leak.

Reply:
Yep. Updated